ClassicAssist/Data/Macros/MacroManager.cs

```diff
@@ -111,7 +111,12 @@ public void Execute( MacroEntry macro )
             {
                 if ( CurrentMacro != null && CurrentMacro.IsRunning )
                 {
-                    if ( CurrentMacro.DoNotAutoInterrupt )
+                    if ( !CurrentMacro.Disableable )
+                    {
+                        return;
+                    }
+
+                    if ( macro == CurrentMacro && macro.DoNotAutoInterrupt )
```

Comment:
Removing !Replay breaks 'Replay()' functionality on macros marked 'DoNotAutoInterrupt', they will end instead of replaying

Why is that wanted!?!?